cli/glnexus_cli.cc

```diff
@@ -666,16 +675,14 @@ int main_discover_alleles(int argc, char *argv[]) {
     console->info() << "discovering alleles in " << ranges.size() << " range(s)";
     vector<GLnexus::discovered_alleles> valleles;
     H("discover alleles", svc->discover_alleles(sampleset, ranges, valleles));
-    unsigned discovered_allele_count=0;
-    for (const auto& dsals : valleles) {
-        discovered_allele_count += dsals.size();
-    }
-    console->info() << "discovered " << discovered_allele_count << " alleles";
+
+    GLnexus::discovered_alleles dsals;
+    utils::merge_all(valleles, dsals);
```

Comment:
unchecked Status -- `merge_all` is quite brief and used only here, perhaps we can just inline it?